index.js

```diff
@@ -249,6 +290,12 @@ var FirefoxBrowser = function (id, baseBrowserDecorator, args) {
         // Ignore failure -- the browser process might have already been
         // terminated.
       }
+    } else if (isWsl) {
+      const tasklist = extractPids (safeExecSync ('tasklist.exe /FI "IMAGENAME eq firefox.exe" /FO CSV /NH /SVC'))
+        .filter(pid => browserProcessPidWsl.indexOf(pid) === -1)
+
+      log.warn ('Killing the following PIDs: %s', tasklist)
```

Comment:
I guess this should be `log.info` too. It seems a bit odd to display warning when everything is operating as expected.